frontend/src/containers/CreateEditPost.tsx

```diff
@@ -91,13 +96,11 @@ function CreateEditPost(props: PropsType) {
                 },
               ];
             });
-            // setCartPlaceList([
-            //   ...initialCartPlaceList,
-            //   {
-            //     place: place,
-            //     day: 0,
-            //   },
-            // ]);
+          });
+          response.data.posts.forEach((route: SimplePostType) => {
+            setCartRouteList((prevState: any) => {
```

Comment:
What did you intend here? Maybe all the routes will be added to `my route section` instead of just one. Did you intend to add them all?

And may be this code can be rewritten like this.
```
let routeList = [...cartRouteList];
response.data.posts?.forEach((route) => routeList = [...routeList, route]);
setCartRouteList(routeList);
```

When `setState` function is called, react re-renders the components.
So your code would make unnecessary rendering.

Reply:
1. `setCartRouteList` function is different from `setRouteList` function!
It sets all the posts saved in the cart so that user can select which post he/she should use when they choose base routes in the route select modal. 
2. I will refactor my codes similar to your suggestions by the way, thank you!

Reply:
1. My question was, is it possible to add multiple posts in one cart? As I know, we decided that a user can add only one post per one cart. Tell me if i know something wrong.

Reply:
1. oh as far as i know the user can add several posts in one cart!